.github/workflows/laboratory-operator.yml

```diff
@@ -0,0 +1,62 @@
+name: Laboratory Operator
+on: [push]
+jobs:
+   test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x]
+        platform: [ubuntu-latest]
+    runs-on: ${{ matrix.platform }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Test
+      working-directory: operators/labInstance-operator/
+      run: |
+         go test ./pkg/
+   build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: LabOperator - KubeScore Check
+      run: |
+       docker run -v  ${{ github.workspace }}/operators/labInstance-operator/:/labInstance-operator zegl/kube-score score --ignore-test=pod-networkpolicy /labInstance-operator/k8s/operator/deployment.yaml
+
+    - name: Build and Push Laboratory-Operator image
```

Comment:
I would add (Dev) to the name for the sake of clarity (as in the frontend file).